src/compiler/parser.rs

```diff
@@ -801,15 +820,18 @@ impl ParserType {
         let mut list: Vec<Node> = vec![];
         let list_pos = self.get_node_pos();
 
+        let mut comma_start: isize = -1;
```

Comment:
You can also do this in a way where you avoid the type annotation on the variable:

```rs
let mut comma_start = -1_isize;
```

I'm not really sure which I prefer 🤷🏻

Reply:
Ah ok ya that numeric-type-suffix reads a little strangely to me